server/api/images/upload.post.ts

```diff
@@ -0,0 +1,9 @@
+export default eventHandler(async (event) => {
+  return hubBlob().handleUpload(event, {
+    multiple: false,
+    ensure: {
+      maxSize: '8MB',
+      types: ['image/png', 'image/jpeg']
```

Comment:
It would be good to allow gifs and webp also.

Reply:
this file and every file in the app directory was autogenerated from the nuxthub starter. this code uses cloudflare's serverless storage so I'll delete this and the autogenerated files when this sprint ends. we'll be storing files in supabase which will allow gif/webp